server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/SelenideConfig.java

```diff
@@ -30,9 +30,9 @@
 public class SelenideConfig {
 
   private enum Browser {
-    firefox("(v46 and lower)"),
-    marionette("(recent Firefox, require Geckodriver)"),
-    chrome("(require Chromedriver)");
+    FIREFOX("(v46 and lower)"),
+    MARIONETTE("(recent Firefox, require Geckodriver)"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00115)